README.md

```diff
@@ -102,6 +122,7 @@ adapter patterns in use today.
     BindExpression[Yield] :
         LeftHandSideExpression[?Yield] :: [lookahead ≠ new] MemberExpression[?Yield]
         :: MemberExpression[?Yield]
+        LeftHandSideExpression[?Yield] :: new
```

Comment:
Okay. I see now. I had to look at the spec to see that the above is a valid MemberExpression.